README.md

```diff
@@ -663,6 +676,36 @@ There is a specific use case where you want to show instructions for different o
 </OperatingSystemTabs>
 ```
 
+### CodeCellPlaceholder
+
+This component only works in notebooks. Notebook code cells are always at the
+top-level of content, but sometimes you'll want to have them nested in other
+components, such as in tabs or in a list. To use this component, add a tag
+starting with `id-` to the cell you'd like to move, then add a
+`<CodeCellPlaceholder tag="id-tag" />` component with the same tag somewhere in
+your markdown. This will move that code cell into the place of the component.
+
+You can then use this component anywhere in your mdx. While you can move code
+cells anywhere, try to keep them relatively close to their position in the
+notebook and preserve their order to avoid confusion.
+
+```mdx
+Here's a list where the second list element is a code cell.
+The cell with tag "id-move-to-list" will be moved inside this list.
```

Comment:
It would probably be more clear to show an actual Jupyter notebook, like the JSON for it, with the two code blocks